src/agent/input-tester/src/test_result/mod.rs

```diff
@@ -343,17 +343,11 @@ impl ExitStatus {
     }
 
     pub fn is_normal_exit(&self) -> bool {
-        match self {
-            ExitStatus::Code(_) => true,
-            _ => false,
-        }
+        matches!(self, ExitStatus::Code(_))
```

Comment:
I'm not sure I like this change.

Reply:
I'm fine with this type, where we are matching against one variant.

Reply:
leaving this one, since it's a single use variant